translator/tocwconfig/sampleConfig/amp_config_linux.yaml

```diff
@@ -1,168 +1,166 @@
 exporters:
-  awscloudwatch:
-    drop_original_metrics:
-      CPU_USAGE_IDLE: true
-      cpu_time_active: true
-    force_flush_interval: 1m0s
-    max_datums_per_call: 1000
-    max_values_per_datum: 150
-    middleware: agenthealth/metrics
-    namespace: CWAgent
-    region: us-west-2
-    resource_to_telemetry_conversion:
-      enabled: true
-    rollup_dimensions:
-    - - ImageId
-    - - InstanceId
-      - InstanceType
-    - - d1
-    - [ ]
-  prometheusremotewrite/amp:
-    add_metric_suffixes: true
-    auth:
-      authenticator: sigv4auth
-    compression: ""
-    disable_keep_alives: false
-    endpoint: https://aps-workspaces.us-west-2.amazonaws.com/workspaces/ws-12345/api/v1/remote_write
-    export_created_metric:
-      enabled: false
-    http2_ping_timeout: 0s
-    http2_read_idle_timeout: 0s
-    max_batch_size_bytes: 3000000
-    namespace: ""
-    proxy_url: ""
-    read_buffer_size: 0
-    remote_write_queue:
-      enabled: true
-      num_consumers: 5
-      queue_size: 10000
-    resource_to_telemetry_conversion:
-      clear_after_copy: true
-      enabled: true
-    retry_on_failure:
-      enabled: true
-      initial_interval: 50ms
-      randomization_factor: 0.5
-      multiplier: 1.5
-      max_interval: 30s
-      max_elapsed_time: 5m0s
-    send_metadata: false
-    target_info:
-      enabled: true
-    timeout: 5s
-    tls:
-      ca_file: ""
-      cert_file: ""
-      include_system_ca_certs_pool: false
-      insecure: false
-      insecure_skip_verify: false
-      key_file: ""
-      max_version: ""
-      min_version: ""
-      reload_interval: 0s
-      server_name_override: ""
-    write_buffer_size: 524288
+    awscloudwatch:
```

Comment:
The indentation is because of our default indentation which was altered previously and this is correcting it. You can ignore the whitespace